zipkin-ui/js/page/trace.js

```diff
@@ -24,9 +25,10 @@ const TracePageComponent = component(function TracePage() {
         totalServices: $('.trace-details.services span').length
       });
       FullPageSpinnerUI.attachTo('#fullPageSpinner');
+      JsonPanelUI.attachTo('#jsonPanel');
       ServiceFilterSearchUI.attachTo('#serviceFilterSearch');
       SpanPanelUI.attachTo('#spanPanel');
-      TraceUI.attachTo('#trace-container');
+      TraceUI.attachTo('#trace-container', {raw: modelview.raw});
```

Comment:
The parameter that is called "modelview" today could be renamed to "data", and then "data" would contain two properties: "modelview" and "raw"?